engine.py

```diff
@@ -394,11 +441,12 @@ def show_panel(self, panel_id, title, bundle, widget_class, *args,
                         panel_interface = interface
                         break
             except hou.OperationFailed:
-                # likely due to panels file not being a valid file, missing, etc. 
+                # likely due to panels file not being a valid file, missing, etc.
                 # hopefully not the case, but try to continue gracefully.
                 self.logger.warning(
-                    "Unable to find interface for panel '%s' in file: %s" % 
-                    (panel_id, self._panels_file))
+                    "Unable to find interface for panel '%s' in file: %s"
+                    % (panel_id, self._panels_file)
```

Comment:
No biggy, I'll let you decide whether you want to do a find and replace and switch all `self.log_*` that use `%` to more logging style `self.logger.*` API without `%`

```suggestion
                    "Unable to find interface for panel '%s' in file: %s",
                    panel_id, self._panels_file
```